src/App.js

```diff
@@ -29,12 +30,21 @@ function App() {
         setTodoList([...todoList, newTodo]);
     } 
 
+    const removeTodo = (id) => {
+     
+      const newListItem = todoList.filter(
+        (TodoListItem) => id !== TodoListItem.id
```

Comment:
I think that the name `TodoListItem` can be simplified to `todoItem` and it does not need to be capitalized.